Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Pull down a Jekyll base image (which uses ruby:2.7.1-alpine3.11)
 
 # dev-stage
-FROM jekyll/jekyll:4.0
+FROM jekyll/jekyll:4.2.2
```

Comment:
@dpeachpeach Did you try it on Apple M1? It seems not working correctly on arm64. Thanks!